client/src/actions/ConfigurationsActions.ts

```diff
@@ -299,7 +328,7 @@ class ConfigurationsActions extends AbstractActions implements IConfigurationsAc
     for (var i in parsedString) {
       if (typeof parsedString[i] === 'string') {
         if (parsedString[i].substring(0, 8) === 'function') {
-          eval('obj[i] = ' + parsedString[i] ); /* tslint:disable-line */
+          global['eval']('obj[i] = ' + parsedString[i] );
```

Comment:
I wanted to remove ` /* tslint:disable-line */` to make sure there are no linting omitions. 
Plus, eval is forbidden, and we still want to use it. This seamed like the best approach, not sure about how since it is not a best practice to use it...